server/form-pages/assess/matchingInformation/matchingInformationTask/matchingInformation.test.ts

```diff
@@ -40,6 +42,13 @@ describe('MatchingInformation', () => {
       expect(page.body).toEqual(defaultArguments)
     })
 
+    it('should set isWheelchairDesignated to essential if undefined but needsWheelchair is yes', () => {
+      ;(retrieveOptionalQuestionResponseFromFormArtifact as jest.Mock).mockReturnValue('yes')
```

Comment:
would it more helpful here to be more explicit in our mock? I think to understand how the question posed in the application:

> Does the person require the use of a wheelchair: Yes | No

maps to the assessment form's `isWheelchairDesignated: 'essential'` I'd like to see the specific message to which `retrieveQuestionResponseFromFormArtifact()`  is replying `'yes'`

i.e. something along the lines of: 
```ts
mockedWith(
  mockApplication,
  mockMatchingInformation,
  'needsWheelchair',
)
```

Reply:
Yup - I wondered about using [jest-when](https://github.com/timkindberg/jest-when) for this, which we used on Accredited Programmes quite a bit but isn't included in Approved Premises. Not sure if there's a way to do this without that, but it might be worth adding it for precision?

Reply:
yes I would support that if you feel it's the best way to get that added clarity 👍